stable_baselines3/common/noise.py

```diff
@@ -84,3 +85,82 @@ def reset(self) -> None:
 
     def __repr__(self) -> str:
         return f'OrnsteinUhlenbeckActionNoise(mu={self._mu}, sigma={self._sigma})'
+
+
+class VectorizedActionNoise(ActionNoise):
+    """
+    A Vectorized action noise for parallel environments.
+
+    :param base_noise: ActionNoise The noise generator to use
+    :param n_envs: (int) The number of parallel environments
+    """
+
+    def __init__(self, base_noise: ActionNoise, n_envs: int):
+        try:
+            self.n_envs = int(n_envs)
+            assert self.n_envs > 0
+        except (TypeError, AssertionError):
+            raise ValueError("Expected n_envs to be positive integer greater than 0")
```

Comment:
```suggestion
            raise ValueError(f"Expected n_envs={n_envs} to be positive integer greater than 0")
```
minor: give the value that was passed